packages/docs/page-config/ui-elements/slider/examples/TrackLabel.vue

```diff
@@ -1,12 +1,12 @@
 <template>
```

Comment:
Doesn't seem to fix it.

<img width="613" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/4df10701-2008-40e0-9c75-529dca64b060">

Reply:
yep, my bad. missed the second slider. fixed

Reply:
Ok, so we fixed the docs example. We didn't fix the underlying issue.

Range should be operational even if min/max are swapped. It's reasonable to expect user would do something similar in app.

We could either fix it in this PR or move to another issue, but this has to be addressed.

Reply:
ok. I think it's right to have fix in this pr. I'll check what we can do.

Reply:
I've created a possible solution,  I can't say how it's optimal as it was not a trivial task to begin with )

Reply:
story is kind of draft not intended for merge,  maybe will be helpful to play around with sliders